pkg/agent/datamodel/types.go

```diff
@@ -1,1899 +1,1899 @@
-	// Valid values are AlwaysAllow and Webhook.
-	// Webhook mode uses the SubjectAccessReview API to determine authorization.
-	// +optional
-	Mode KubeletAuthorizationMode `json:"mode,omitempty"`
-
-	// webhook contains settings related to Webhook authorization.
-	// +optional
-	Webhook KubeletWebhookAuthorization `json:"webhook"`
-}
-
-type KubeletAuthorizationMode string
-
-type KubeletWebhookAuthorization struct {
-	// cacheAuthorizedTTL is the duration to cache 'authorized' responses from the webhook authorizer.
-	// +optional
-	CacheAuthorizedTTL Duration `json:"cacheAuthorizedTTL,omitempty"`
-	// cacheUnauthorizedTTL is the duration to cache 'unauthorized' responses from the webhook authorizer.
-	// +optional
-	CacheUnauthorizedTTL Duration `json:"cacheUnauthorizedTTL,omitempty"`
-}
-type CSEStatus struct {
-	// ExitCode stores the exitCode from CSE output.
-	ExitCode string `json:"exitCode,omitempty"`
-	// Output stores the output from CSE output.
-	Output string `json:"output,omitempty"`
-	// Error stores the error from CSE output.
-	Error string `json:"error,omitempty"`
-	// ExecDuration stores the execDuration in seconds from CSE output.
-	ExecDuration string `json:"execDuration,omitempty"`
-	// KernelStartTime of current boot, output from systemctl show -p KernelTimestamp
-	KernelStartTime string `json:"kernelStartTime,omitempty"`
-	// SystemdSummary of current boot, output from systemd-analyze
-	SystemdSummary string `json:"systemdSummary,omitempty"`
-	// CSEStartTime indicate starttime of CSE
-	CSEStartTime string `json:"cseStartTime,omitempty"`
-	// GuestAgentStartTime indicate starttime of GuestAgent, output from systemctl show walinuxagent.service -p ExecMainStartTimestamp
-	GuestAgentStartTime string `json:"guestAgentStartTime,omitempty"`
-	// BootDatapoints contains datapoints (key-value pair) from VM boot process.
-	BootDatapoints map[string]string `json:"bootDatapoints,omitempty"`
-}
-
-type CSEStatusParsingErrorCode string
-
-const (
-	// CSEMessageUnmarshalError is the error code for unmarshal cse message
-	CSEMessageUnmarshalError CSEStatusParsingErrorCode = "CSEMessageUnmarshalError"
-	// CSEMessageExitCodeEmptyError is the error code for empty cse message exit code
-	CSEMessageExitCodeEmptyError CSEStatusParsingErrorCode = "CSEMessageExitCodeEmptyError"
-	// InvalidCSEMessage is the error code for cse invalid message
-	InvalidCSEMessage CSEStatusParsingErrorCode = "InvalidCSEMessage"
-)
-
-type CSEStatusParsingError struct {
-	Code    CSEStatusParsingErrorCode
-	Message string
-}
-
-func NewError(code CSEStatusParsingErrorCode, message string) *CSEStatusParsingError {
-	return &CSEStatusParsingError{Code: code, Message: message}
-}
-
-func (err *CSEStatusParsingError) Error() string {
-	return fmt.Sprintf("CSE has invalid message=%q, InstanceErrorCode=%s", err.Message, err.Code)
-}
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.

```

Comment:
pls `git checkout master -- pkg/agent/datamodel`

guessing it's line endings or something but hard to eyeball this

Reply:
not resolved?

Reply:
well... That is confusing. When I look at https://github.com/Azure/AgentBaker/pull/2111/files _pkg/agent/datamodel_ doesn't show up. But it's in the commit...

And git checkout master -- pkg/agent/datamodel/types.go after making sure master is current doesn't pick up a change

This doesn't spot it either... git diff --name-status master..deeperLoggingCSE

Reply:
not confident if we want to continue to argue with git on this, or start a fresh pr and avoid this :\